metadata/Cargo.toml

```diff
@@ -19,12 +19,14 @@ ink_prelude = { version = "2.1.0", path = "../prelude/", default-features = fals
 ink_primitives = { version = "2.1.0", path = "../primitives/", default-features = false }
 
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
+serde_json = "1.0"
 derive_more = { version = "0.99", default-features = false, features = ["from"] }
-scale-info = { version = "0.2", default-features = false, features = ["derive"] }
+scale-info = { version = "0.3", default-features = false, features = ["derive"] }
+semver = { version = "0.10.0", features = ["serde"] }
+url = { version = "2.1.1", features = ["serde"] }
```

Comment:
What are these (and `serde_json`) new dependencies good for?

Reply:
Good for nothing, a hangover from when the extended types were defined here. Good catch!

Reply:
`serde_json` should probably be a `dev-dependency` again.

Reply:
Yeah I've moved it back, my commits have not made it through to this PR yet...